### ISSUE TITLE
Question about Batch Normalization?

### ISSUE THREAD

**Issue by Jingyilang** [chqiwang/convseg#5]
Thanks for your code. But I have a question about the implementation af Batch Normalization.

`if use_bn:
                w = layers.batch_norm(inputs=v, decay=0.9, is_training=is_train, center=True, scale=True,
                                      scope='BatchNorm_w_%d' % i)
                v = layers.batch_norm(inputs=w, decay=0.9, is_training=is_train, center=True, scale=True,
                                      scope='BatchNorm_v_%d' % i)`
 Where w is computed by the inputs=v and v is computed by the inputs=w. Is there a mistake?  The correct code I think is
`if use_bn:
                w = layers.batch_norm(inputs=w, decay=0.9, is_training=is_train, center=True, scale=True,
                                      scope='BatchNorm_w_%d' % i)
                v = layers.batch_norm(inputs=v, decay=0.9, is_training=is_train, center=True, scale=True,
                                      scope='BatchNorm_v_%d' % i)`
Is it true?

**Comment by chqiwang** [comment]
yes! It’s a mistake. thanks for point it out.
